test/test_derivatives.jl

```diff
@@ -16,6 +16,9 @@ else
     end
 end
 
+const datadir = joinpath(Pkg.dir("Celeste"), "test", "data")
```

Comment:
Yes, just remove this. `datadir` is already defined in `runtests.jl`. All the `test_*.jl` files are included into that file, so they'll be in the same namespace/scope as `datadir` already: including just a subset of the `test_*.jl` files doesn't affect this.